frontend/www/js/omegaup/login/signin.ts

```diff
@@ -26,7 +26,7 @@ OmegaUp.on('ready', () => {
   function redirect(isAccountCreation: boolean): void {
     const params = new URL(document.location.toString()).searchParams;
     const pathname = params.get('redirect');
-    if (pathname && pathname.indexOf('/') !== 0) {
+    if (pathname) {
```

Comment:
```suggestion
    if (pathname && pathname.indexOf('/') === 0) {
```

la intención de esto es que si alguien te da un URL con `redirect=http://mipaginamaligna.com`, no sigamos ese redirect. pero parece que pusimos la condición al revés ^^;

Reply:
Ahh, y eso ocasionaba que sólo funcionara la redirección en con el patrón incorrecto, hasta que se corrigió esa url en la migración de ArenaProblem.